### PR TITLE
docs: move and update beta list samples for video

### PR DESCRIPTION
**Pull request by nnegrey**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-automl/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/beta/list-datasets.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      console.log(`\tseconds ${dataset.createTime.seconds}`);
+      console.log(`\tnanos ${dataset.createTime.nanos / 1e9}`);
+
+      // [END automl_video_object_tracking_list_datasets_beta]
+      console.log(
+        `Video classification dataset metadata: ${dataset.videoClassificationDatasetMetadata}`
+      );
+      // [END automl_video_classification_list_datasets_beta]
+
+      // [START automl_video_object_tracking_list_datasets_beta]
```
Comment (JustinBeckwith) [inline samples/beta/list-datasets.js]: This feels like a weird place to start a snippet.  Is that intentional?

File: samples/beta/list-datasets.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      console.log(
+        `Video object tracking dataset metadata: ${dataset.videoObjectTrackingDatasetMetadata}`
+      );
+      // [START automl_video_classification_list_datasets_beta]
```
Comment (JustinBeckwith) [inline samples/beta/list-datasets.js]: Ditto, thinking maybe these don't belong here :)

File: samples/beta/list-datasets.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(projectId = 'YOUR_PROJECT_ID', location = 'us-central1') {
+  // [START automl_video_classification_list_datasets_beta]
```
Comment (JustinBeckwith) [inline samples/beta/list-datasets.js]: For my own understanding - are these features available in the GA version of the API? Or only in the beta?

**Comment by bcoe** [comment]
@nnegrey happy to help land these updates (I saw a few similar PRs) once @JustinBeckwith's comments are addressed. Please feel free to flag me once you're ready.

File: samples/beta/list-datasets.js (hunk 1 above)
Comment (nnegrey) [reply samples/beta/list-datasets.js]: Yep, the `START` and `END` pieces allow us to show one piece on the docs while keeping the majority of the sample the same. 

One file: https://github.com/googleapis/nodejs-automl/blob/master/samples/list_datasets.js


Docs:
1. https://cloud.google.com/natural-language/automl/docs/manage-datasets#automl_language_text_classification_list_datasets-nodejs
2. https://cloud.google.com/natural-language/automl/docs/manage-datasets#automl_language_text_classification_list_datasets-nodejs
3. https://cloud.google.com/natural-language/automl/docs/manage-datasets#automl_language_text_classification_list_datasets-nodejs

File: samples/beta/list-datasets.js (hunk 3 above)
Comment (nnegrey) [reply samples/beta/list-datasets.js]: The video flavor of AutoML is in beta along with Tables

But these other versions are in GA:
- Language: Entity Extraction, Sentiment Analysis, Classification
- Vision: Object Detection, Classification
- Translate

**Pull request merged**